internal/app/parcel/service.go

```diff
@@ -12,12 +12,23 @@ type service struct {
 	repo svc.ParcelRepository
 }
 
+// NewService is to generate for new repo
 func NewService(repo svc.ParcelRepository) *service {
 	return &service{
 		repo: repo,
 	}
 }
 
+// get Parcel List
+func (s *service) GetParcels(ctx context.Context, status int, limit int, offset int) ([]model.Parcel, error) {
+	parcel, err := s.repo.GetParcelsList(ctx, status, limit, offset)
+
+	if err != nil {
```

Comment:
No need to check error here. This error is coming from repository, just return the output of the repo to handler.

Reply:
done